BLSQ/iaso
# POLIO-1657 Chronogram: make `user_in_charge` a `CharField` instead of a `ForeignKey`

Chronogram: make `user_in_charge` a `CharField` instead of a `ForeignKey`.

Related JIRA tickets : [POLIO-1657](https://bluesquare.atlassian.net/browse/POLIO-1657)

## Changes

`user_in_charge` is now a `CharField` instead of a `ForeignKey`.

## How to test

1. Go to `http://localhost:8081/dashboard/polio/chronogram/`
2. Go to the detail of a Chronogram
3. create a new task or edit an existing one
    - the "Responsible" field should be an open text field

## Print screen

![responsible](https://github.com/user-attachments/assets/7710d008-8933-41c3-b643-aaf36cd9f5fb)


[POLIO-1657]: https://bluesquare.atlassian.net/browse/POLIO-1657?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
